.woodpecker.yml

```diff
@@ -1,5 +1,10 @@
 variables:
   - &cd_workdir "cd /usr/src/app"
+  - &rust_image "rustlang/rust:nightly-bookworm"
+  - install_binstall: &install_binstall
```

Comment:
You could do what the Dockerfile is currently doing for this step:

```
wget -O- https://github.com/cargo-bins/cargo-binstall/releases/latest/download/cargo-binstall-x86_64-unknown-linux-musl.tgz | tar -xvz -C /usr/local/cargo/bin
```

This will also eliminate the need to copy `cargo-binstall` to `/usr/local/cargo/bin` since it will be extracted there directly.

Reply:
Sweet, I'll try it out.